frontend/src/hooks/auth/index.js

```diff
@@ -0,0 +1,3 @@
+import isLoggedIn from "./useIsLoggedIn";
+import logout from "./logout";
+export default { isLoggedIn, logout };
```

Comment:
> Hooks should be prefixed with `use`, otherwise it can be confused with a normal function.

I am aware of this, but here we wanted a structure like auth.isLogedIn.